vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -1030,33 +1030,49 @@ public View getChildView(int groupPosition, int childPosition, boolean isLastChi
         final ImageView thumbView = convertView.findViewById(R.id.filtered_list_avatar);
         final TextView nameTextView = convertView.findViewById(R.id.filtered_list_name);
         final TextView domainNameTextView = convertView.findViewById(R.id.filtered_list_domain);
-        final TextView statusTextView = convertView.findViewById(R.id.filtered_list_email);
+        final TextView emailTextView = convertView.findViewById(R.id.filtered_list_email);
+        final ImageView onlineStatusImageView = convertView.findViewById(R.id.filtered_list_online_status);
 
         // reported by GA
         // it should never happen but it happened...
-        if ((null == thumbView) || (null == nameTextView) || (null == statusTextView)) {
+        if ((null == thumbView) || (null == nameTextView) || (null == emailTextView)) {
             Log.e(LOG_TAG, "## getChildView() : some ui items are null");
             return convertView;
         }
 
-        // Contacts not in priority are seen different
-        if (!participant.isViewedInPriority()){
-            //final int semiTransparentGrey = Color.argb(155, 185, 185, 185);
-            //thumbView.setAlpha( 0.5f);
+        // No Tchap users are displayed differently than Tchap users
+        if (!participant.isMatrixUser()){
             nameTextView.setTypeface(null, Typeface.ITALIC);
         } else {
-            //thumbView.clearColorFilter();
+            onlineStatusImageView.setVisibility(VectorUtils.isUserOnline(mContext, mSession, participant.mUserId, null) ? View.VISIBLE : View.GONE);
+            domainNameTextView.setPadding(0, 30, 0, 0);
+            nameTextView.setPadding(0, 18, 0, 0);
```

Comment:
Is this padding used to center vertically the display name when emailTextView is Gone?